collect_app/src/main/java/org/odk/collect/android/formmanagement/ServerFormDownloader.java

```diff
@@ -0,0 +1,30 @@
+package org.odk.collect.android.formmanagement;
+
+import org.odk.collect.android.R;
+import org.odk.collect.android.application.Collect;
+import org.odk.collect.android.utilities.MultiFormDownloader;
+
+import java.util.Collections;
+import java.util.HashMap;
+
+/**
+ * Provides a sarcophagus for {@link org.odk.collect.android.utilities.MultiFormDownloader} so it
```

Comment:
Hah. Never seen sarcophagus used that way but it makes so much sense.

Reply:
It's great terminology. I like that it implies it is protecting both the thing in the "sarcophagus" and the things outside.